lib/scrum_board.rb

```diff
@@ -26,7 +26,7 @@ def done_column
   def accepted_column
     accepted_columns = columns.select{|c| c.name =~ @settings.accepted_column_name_regex }
     if accepted_columns.empty?
-      EmptyColumn.new
+      EmptyColumn.new(@settings)
```

Comment:
If you use a default value of ```nil``` as per the previous comment, you can skip this.

Reply:
@jimmykarily why? it could be different from `nil` or am I missing something?

Reply:
I can't remember why I wrote this but looking at it now, I have to agree with you. Ignore my comment.